### ISSUE TITLE
Can= Can_o_nosleep do not work like expected

### ISSUE THREAD

**Issue by Jhobean** [Sphereserver/Source-X#956]
How to reproduce:

Start a server with sleepsector activate at 1 min and go in any city. Launch the below function "TEST". After awaking the sector (Go 1,1 and leaving) the item should trig each 2 second for eternity.

On correct behavior, Item should continu to tick after the sector is awake and get asleep again.

```
[itemdef i_test]
id=i_gold
type=t_eq_script
Name=Test de sleep
can=can_o_nosleep

on=@create
timer=2
serv.log the UI of the test timer is <uid>

on=@timer
serv.log I ticking!! My sector sleep?<sector.issleeping> cansleep? <sector.cansleep>
timer=2
return 1

[FUNCTION test]
serv.newitem i_test
new.p=1,1
```

**Comment by Jhobean** [comment]
Ok I made some test:

On the commit: [0b1e59c8 ](https://github.com/Jhobean/Source-X/commit/0b1e59c801d23db529080ee46b8b933ee7590054)
CAN_O_NOSLEEP is implemented. The flag seem to work but its a false impression. Sector juste NEVER fall asleep again. The commit implemented a BUG fixed dew week later in this commit: https://github.com/Jhobean/Source-X/commit/50ad5b386305f4a8e71fe4a1f8366a2862130fa4

In conclusion CAN_O_NOSLEEP just dont work or I use it not correctly.

**Comment by Jhobean** [comment]
If we achieve to fix this CAN_O_NOSLEEP. We should implement this feature in parallel of the flag
https://github.com/Jhobean/Source-X/commit/ca673cd6367bb78608bd276c0908ec560c5473bf

**Comment by LuxionUO** [comment]
I can confirm, can_o_nosleep is not working.

**Comment by Jhobean** [comment]
@cbnolok we need your help for this one. I'll pay the beer the night you will work in this 🤣. Just tell me where to ship it.

**Comment by Jhobean** [comment]
Fix here: https://github.com/Sphereserver/Source-X/commit/ffbb62eccfdadc7b75a9625dd1575dfc20262dc4

Thx @cbnolok
